backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -725,4 +725,15 @@ SearchException customIndexSettingsJsonSyntaxErrors(String filePath, @Cause Exce
 	@Message(id = ID_OFFSET + 136, value = "Invalid use of 'missing().use(...)' for a distance sort. Elasticsearch always assumes missing values have a distance of '+Infinity', and this behavior cannot be customized.")
 	SearchException missingAsOnSortNotSupported(@Param EventContext context);
 
+	@Message(id = ID_OFFSET + 137, value = "The index schema named predicate '%1$s' was added twice.")
+	SearchException indexSchemaNamedPredicateNameConflict(String relativeFilterName, @Param EventContext context);
```

Comment:
This method is never called, it can be removed.

Reply:
It is used in: org.hibernate.search.backend.elasticsearch.document.model.dsl.impl.AbstractElasticsearchIndexSchemaObjectNodeBuilder.putNamedPredicate

Reply:
Right, I put the comment on the wrong line. Sorry.